nova/MessageReducer.ts

```diff
@@ -0,0 +1,37 @@
+import { db } from './firebase';
+import { Request, Response } from 'express';
+import { Member } from '../src/DataTypes';
+import { ErrorResponse, MemberResponse, ImageResponse } from '../src/APITypes';
+import { getFilePath } from './utils'
+
+const fs = require('fs');
+
+export let reduceMessage = async function (req: Request, res: Response): Promise<MemberResponse | ImageResponse | ErrorResponse> {
+    if (!req.body.member) {
+        return {
+            status: 400,
+            error: "Couldn't update member with undefined data or undefined image"
+        };
+    }
+    if (req.body.member) {
```

Comment:
I think it might be better if you made a separate function called liked `updateMember` or something like that that actually handles updating the member's information

Reply:
Yes, this looks much cleaner. I've added `updateMember` function.